pkg/deploy/reaper/reaper.go

```diff
@@ -65,6 +66,20 @@ func (reaper *DeploymentConfigReaper) Stop(namespace, name string, timeout time.
 			// Better not error out here...
 			glog.Infof("Cannot delete ReplicationController %s/%s: %v", rc.Namespace, rc.Name, err)
 		}
+
+		// Delete all deployer and hook pods
```

Comment:
I think filtering out only non-complete and non-cancelled deployer pods is an overkill here (I'm not sure if I can use the field selector here anyway). Also I can decide to remove DC in a middle of the deployment process, in that case the "running" deployer pod can be missed, right?